aries-site/src/examples/components/togglegroup/ToggleGroupViews.js

```diff
@@ -47,8 +48,10 @@ export const ToggleGroupViews = () => {
   }, []);
   const [value, setValue] = useState('table');
   return (
-    <>
-      <Heading id="spaceX-rocket-launches">Space X Launch Data</Heading>
+    <ContentPane gap="medium">
+      <Heading id="spaceX-rocket-launches" level={2} margin="none">
+        SpaceX launches
+      </Heading>
```

Comment:
Can we look at the treatment of the cards when "card view" toggle is selected? I think this is a case where we need to add the elevation via props, right?

<img width="829" alt="Screenshot 2025-02-26 at 1 51 54 PM" src="https://github.com/user-attachments/assets/76455a89-967a-4143-8346-e349661aa800" />

Reply:
Fixed. Added `elevation="medium"`